docs/pbc/all/service-point-management/202311.0/unified-commerce/service-points-cart-checkout-feature-overview.md

```diff
@@ -6,10 +6,34 @@ template: concept-topic-template
 
 This feature provides an example of a replacement strategy used by Click & Collect. The strategy is needed to replace a product offer with another product offer after a customer changes a service point or a shipment type in cart.
 
+How does it work?
+* When a customer changes a shipment type and/or a service point, the replacement strategy is triggered.
+* The strategy applies the selected shipment type and/or service point to the product offers in the system. These are available for the selected product SKU and Merchant linked to the product offer.
```

Comment:
Please clarify what you mean by the second sentence here.

Reply:
The SKU and Merchant of the current product offer will be applied as a filter to search the target offer.